src/OpenTK.Compute/OpenCL/CL.cs

```diff
@@ -313,11 +328,15 @@ public static string GetDeviceInfoAsString(IntPtr device, DeviceInfo paramName,
 				return buffer;
 			}
 		}
-		public static unsafe IntPtr CreateBuffer<T>(IntPtr context, MemoryFlags flags, Span<T> array, out CLResultCode errorCode) where T : unmanaged
+
+		/// <summary>
+		/// Introduced in OpenCL 1.0
+		/// </summary>
+		public static unsafe IntPtr CreateBuffer<T>(IntPtr context, MemoryFlags flags, Span<T> span, out CLResultCode errorCode) where T : unmanaged
 		{
-			fixed (T* b = array)
+			fixed (T* b = span)
 			{
-				IntPtr buffer = CreateBuffer(context, flags, (UIntPtr)(Marshal.SizeOf<T>() * array.Length), (IntPtr)b, out errorCode);
+				IntPtr buffer = CreateBuffer(context, flags, (UIntPtr)(Marshal.SizeOf<T>() * span.Length), (IntPtr)b, out errorCode);
```

Comment:
`Marshal.Sizeof<T>` can be changed to `sizeof(T)`. 
And is possibly more correct due to `Marshal.Sizeof<T>` returning the marshalled size of the struct, but there is no marshalling done.